### PR TITLE
Add a performance-based dashboard for analysing generic service FE & BE

### PR DESCRIPTION
**Pull request by andyloughran**
# Description:

Adds a capacity dashboard for troubleshooting performance tests.

### CODE REVIEW

File: dashboards.tf [hunk 1]
```diff
@@ -398,3 +398,10 @@ module "orange_sla_dashboard" {
   source = "./modules/dashboard"
   path   = "service-level-agreements/orange.json"
 }
+
+### Capacity & Performance Dashboards ### 
+
+module "performance_application" {
+  source = "./modules/dashboard"
+  path = "capacity/application-performance-review.json
```
Comment (chrisdodd93) [inline dashboards.tf:406]: This `path = "capacity/application-performance-review.json` needs to be `path = "capacity/application-performance-review.json"` - Just missing off the " at the end.

File: dashboards.tf [hunk 2]
```diff
@@ -398,3 +398,10 @@ module "orange_sla_dashboard" {
   source = "./modules/dashboard"
   path   = "service-level-agreements/orange.json"
 }
+
+### Capacity & Performance Dashboards ### 
+
+module "performance_application" {
```
Comment (chrisdodd93) [inline dashboards.tf:404]: Just ensure that the formatting is correct here, like the spacing...
```suggestion
module "performance_application" {
  source = "./modules/dashboard"
  path   = "capacity/application-performance-review.json"
}
```

**Review by chrisdodd93 — CHANGES_REQUESTED** [review]

File: dashboards.tf (hunk 1 above)
Comment (andyloughran) [reply dashboards.tf:406]:
```suggestion
  path = "capacity/application-performance-review.json"
```

File: dashboards.tf [hunk 3]
```diff
@@ -475,9 +475,16 @@ module "orange_sla_dashboard" {
   path   = "service-level-agreements/orange.json"
 }
 
+
+### Capacity & Performance Dashboards ### 
+
+module "performance_application" {
+  source = "./modules/dashboard"
+  path   = "capacity/application-performance-review.json"
```
Comment (chrisdodd93) [inline dashboards.tf:483]:
```suggestion
  path   = "capacity/application-performance-review.json"
}
```

**Review by chrisdodd93 — CHANGES_REQUESTED** [review]

**Review by chrisdodd93 — APPROVED** [review]

**Pull request merged**